src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -127,4 +129,26 @@ void regmodule_offline_transformations(py::module m) {
             manager.run_passes(model);
         },
         py::arg("model"));
+
+
+    m_offline_transformations.def(
+        "paged_attention_transformation",
+        [](std::shared_ptr<ov::Model> model) {
+            std::cout << "___CALLING SDPAToPagedAttention___" << std::endl;
+            ov::pass::Manager manager;
+            manager.register_pass<ov::pass::SDPAToPagedAttention>();
+            manager.run_passes(model);
+            std::cout << "___AFTER CALLING SDPAToPagedAttention___" << std::endl;
+        },
+        py::arg("model"));
+
+    m_offline_transformations.def(
```

Comment:
debug code, I believe it's better to move it to the separate PR if we want to move PrintModel from CPU transformations to common folder and create python binding for it.

Reply:
fixed